### PR TITLE
Added works about sections

### PR DESCRIPTION
**Pull request by gfrancoarq**
### I've added the Works and About section. It includes content, assets, and styling.
* These are he mobile website sections where I will list my portfolio projects and skills.
* These changes complete the "Portfolio: setup and mobile first" activity

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -70,7 +74,275 @@ <h2>I'm a software developer</h2>
...
+
+    <!-- Project 2 Portfolio: finish mobile version START -->
+
+    <!-- WORKS SECTION START -->
+    <section class="header-content">
+      <!-- SECTION TITLE START -->
+      <div class="section-title">
+        <h1 class="recent-works">My Recent Works</h1>
+      </div>
+      <img class="indicator" src="images/Indicator.svg" alt="" />
```
Comment (danmainah) [inline index.html:90]: - [ ] Kindly add your image indicator as a background image, as a decorating image the indicator should be added as a background image and not using img tag.

File: index.html [hunk 2]
```diff
@@ -70,7 +74,275 @@ <h2>I'm a software developer</h2>
...
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <button type="button" class="resume-button hoverme">Get My Resume</button>
     </section>
+    <!-- ABOUT ME END -->
+
+    <!-- SKILLCARD CONTAINER START -->
+    <section class="skillcard-container">
```
Comment (danmainah) [inline index.html:259]: - [ ] @gfrancoarq great work, but you should not separate the skill sets with your about section, they belong to the same section, kindly remove this section tag.

File: index.html [hunk 3]
```diff
@@ -70,7 +74,275 @@ <h2>I'm a software developer</h2>
...
+          <li></li>
+          <li></li>
+          <li></li>
+        </ul>
+        <div class="tags">
+          <p class="skill-tag">JavaScript</p>
+          <p class="skill-tag">Ruby</p>
+          <p class="skill-tag">Html</p>
+          <p class="skill-tag">CSS</p>
+        </div>
```
Comment (danmainah) [inline index.html:281]: - [ ] Kindly use `ul` and `li` to list your technologies, this is a project requirement.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @gfrancoarq , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by prolajumokeoni — APPROVED** [review]
Hi @gfrancoarq 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**